test/VoidMethExceptionsTest.thrift

```diff
@@ -0,0 +1,13 @@
+namespace java thrift.test.voidmethexceptions
+
+exception TExampleException {
+  1: required string message;
+}
+
+service TAppService01 {
+  string returnString(1: string msg, 2: bool throwException) throws (1:TExampleException error);
+  void returnVoidThrows(1: string msg, 2: bool throwException) throws (1:TExampleException error);
+  void returnVoidNoThrowsRuntimeException(1: string msg, 2: bool throwException);
+  void returnVoidNoThrowsTApplicationException(1: string msg, 2: bool throwException);
+  oneway void onewayVoidNoThrows(1: string msg, 2: bool throwException);
```

Comment:
Just to be clear: A oneway does not return anything. Including exceptions. There is no return. None.

Reply:
Yes, test checks exactly that (no client-side exceptions when server throws, no client-side wait when server hangs). Method name reflects correct signature for such methods.